resources/assets/js/components/ChannelDropdown.vue

```diff
@@ -70,6 +76,18 @@
                         .startsWith(this.filter.toLocaleLowerCase());
                 });
             }
+        },
+
+        methods: {
+            onCLickOutside() {
```

Comment:
is there a reason why the "L" is capitalized? haha

Reply:
Thanks, I changed that typo but the tests stopped passing.
I think it was "pinned_threads_are_listed_first" test that failed, but on my local everything is passing, and in fact... the build was marked as OK by travis and the typo fix just broke it.

Any idea what could've gone wrong?

Reply:
Just squashed both commits and the builed passed....

I guess travis just had to run again. Something could be going on there since the PHP 7.1 build took over 4 minutes. Twice as the PHP 7.2 build.